### PR TITLE
Add Docker mounts to init script

### PR DESCRIPTION
**Pull request by shrivara**
https://github.com/Shippable/node/issues/117

### CODE REVIEW

File: initScripts/x86_64__Ubuntu_16.04__Docker_17.06.sh [hunk 1]
```diff
@@ -185,7 +185,10 @@ setup_mounts() {
   mkdir -p $REQKICK_DIR
   mkdir -p $BUILD_DIR
 
-  REQPROC_MOUNTS="-v $BASE_DIR:$BASE_DIR"
+  REQPROC_MOUNTS="-v $BASE_DIR:$BASE_DIR \
```
Comment (ric03uec) [inline initScripts/x86_64__Ubuntu_16.04__Docker_17.06.sh]: should be envs on top of this file. individual functions can append to the same variable if required

Comment (shrivara) [reply initScripts/x86_64__Ubuntu_16.04__Docker_17.06.sh]: REQPROC_MOUNTS is defined as `export REQPROC_MOUNTS=""` at the top already. Are you saying this should be:
REQPROC_MOUNTS="$REQPROC_MOUNTS -v ..."?

Comment (ric03uec) [reply initScripts/x86_64__Ubuntu_16.04__Docker_17.06.sh]: yes

Comment (shrivara) [reply initScripts/x86_64__Ubuntu_16.04__Docker_17.06.sh]: Updated this and a few other places.

**Pull request merged**